ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml

```diff
@@ -115,8 +115,8 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-gcp-ipi-cco-manual-users-static
-- as: gcp-ipi-nested-virtualization-f28
-  cron: 55 20 8 * *
+- as: gcp-ipi-nested-virtualization-osdisk-type-size-f28
```

Comment:
shall we update the frequency to `f14` for more coverage?

Reply:
I think `f28` is enough, according to our test history. Thanks! 

Test Case | Last 10 Runs Results | Comment
-- | -- | --
[OCP-30790 Install cluster with valid disktype and disksize specified](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-30790) | All Passed| 
[OCP-32001 Install a cluster with nested virtualization enabled](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-32001) | 9 Passed 1 Failed | The failed one was during "Full Functional Testing_OCP_4.9_Installer_Before_GA"